Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN apt-get -qq update \
 	&& rm -rf /var/lib/apt/lists/*
 
 # Download and install hugo
-ENV HUGO_VERSION 0.53
+ENV HUGO_VERSION 0.54
```

Comment:
in my experience, it needs to be 0.54.0 for this one

Reply:
thank you @omniwired